locallibrary/catalog/models.py

```diff
@@ -1,3 +1,78 @@
 from django.db import models
+from django.urls import reverse
+import uuid
 
 # Create your models here.
+class Genre(models.Model):
+    """Model representing a book genre."""
+    name = models.CharField(max_length=200, help_text='Enter a book genre (e.g. Science Fiction)')
```

Comment:
Nội dung của option `help_text` cần đc translate phục vụ cho tính năng đa ngôn ngữ ứng dụng
https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#internationalization-in-python-code

sửa tương tự trong file nhé